.github/workflows/tests.yaml

```diff
@@ -159,126 +60,26 @@ jobs:
           key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}
           restore-keys: |
             ${{ runner.os }}-go-
-      - run: go get -v -t -d ./...
+      - run: go get -v -t -d -tags ${{ matrix.db_type }} ./...
       - name: Unit tests
         env:
           DRAND_TEST_LOGS: "${{ runner.debug == '1' && 'DEBUG' || 'INFO' }}"
```

Comment:
Should we also add `CI: "true"` to it here?
The CI env variable is making some timeout/sleeps slower in our test suite, which can help with running them on slow machines